deepfence_worker/cronjobs/notification.go

```diff
@@ -89,42 +137,40 @@ func processIntegrationRow(integrationRow postgresql_db.Integration, msg *messag
 	return errors.New("No integration type")
 }
 
-func injectNodeData[T any](results []T, common model.ScanResultsCommon,
+func injectNodeDatamap(results []map[string]interface{}, common model.ScanResultsCommon,
```

Comment:
Why are we modifying this function?
We should work with strongly types data as much as possible